### ISSUE TITLE
[Enhance] Update Accessibility Chart 

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#3644]
**User Story:**
As Spark we want to update our Accessibility Chart so that we are showing colors that can be used with a black background.

**Notes:**
Requestor: SME: RV, Woods Support Room

**Current State:**
Accessibility Chart:
https://sparkdesignsystem.com/using-spark/foundations/color-accessibility
according to that chart, the only color that can be used with a black background is orbit lightest.
UX Issue: https://github.com/sparkdesignsystem/Backlog/issues/1274

**AC:**
- Images, UX [JANE]
- I will see updated updated Accessibility Chart to show colors that can be used with a black background
- Image resolution will match for High Res & Page Image.
- Charts will be consistent with High Res & Page Image.
- Text on High Res & Page Image will be consistent.
